indexing-service/src/main/java/io/druid/indexing/common/task/TaskUtils.java

```diff
@@ -26,8 +26,7 @@
 {
   public static String makeId(String id, final String typeName, String dataSource, Interval interval)
```

Comment:
I think we should just remove TaskUtils class as it only contains one method which is really just calling another method now. All the usage of this method are inside Task classes which can directly use AbstractTask.joinId.